src/pfe/portal/modules/User.js

```diff
@@ -876,13 +852,8 @@ module.exports = class User {
       log.debug("Codewind Docker Registry List: " + JSON.stringify(registrySecretList));
     } catch (err) {
       const msg = "Failed to get the Codewind Docker Config Registries";
-      log.error(err);
       log.error(msg);
-      const retval = {
-        statusCode: 500,
-        body: msg
-      };
-      return retval;
+      throw err;
     }
     
     const retval = {
```

Comment:
Again, this can just return the list. `GET /api/v1/registrysecrets` knows that's a 200 and it makes it easier to re-use this function elsewhere.

Reply:
updated